src/store/Chat/reducer.js

```diff
@@ -1,30 +1,32 @@
 import {
+  LOAD_CHATS_SUCCESS,
+  LOAD_CHATS_FAILURE,
   CHAT_SET_CONFIG,
   CHAT_SET_ACTIVE,
   CHAT_TOGGLE_OPEN,
   OPERATOR_SET_FILTER,
 } from './constants';
 
-import makeDummy from '../dummy';
-
-
-// Flag to enable putting dummy data into redux
-const DUMMY_DATA = true;
```

Comment:
Should we pull this from an environment variable and then when we're just working on the Operator without running the Daemon we can use dummy data, but if we're working on a Daemon+Operator task we can disable it and load from the Daemon? I'm just thinking from a dependencies perspective